extensions/extensions-docker/aissemble-spark-infrastructure/pom.xml

```diff
@@ -88,6 +88,21 @@
                     </execution>
                 </executions>
             </plugin>
```

Comment:
I: This block was oddly in the parent pom but was only ever used here.  This change aligns the declaration and use more effectively.